index.html

```diff
@@ -1,270 +1,341 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body>
+
+    <header class="header-nav">
+        <nav class="navbar fixed-top">
+            <a href="" class="logo">
+                Tite Lunda
+            </a>
+
+            <img src="./Images/Union.png" alt="navigation bars" class="menu">
+
+        </nav>
+
+
+    </header>
+
+    <section class="headline" id="headline">
+
+        <div class="container">
+            <h1 class="identity">
+                Hey there. I'm, Tite Lunda <br><span>I'm a software developer</span>
+            </h1>
+            <p class="profile">
+                <b></b> I can help you build a product , feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+
+            <ul class="icon-group">
+                <li class="icon"> <a href="#https://twitter.com/LundaTite"><i class="fa-brands fa-twitter"></i></a></li>
+                <li class="icon"> <a href="#https://www.linkedin.com/in/tite-lunda-094956199/"><i
+                            class="fa-brands fa-linkedin"></i></a></li>
+                <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
+                <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
+                <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-angellist"></i></a>
+                </li>
+            </ul>
+            <ul class="footer-Icon2">
+
+
+                <li class="Icons">
+                    <a href="#https://github.com/TiteLunda"><img src="./Images/github.png" alt="twitter"></a>
+                </li>
+                <li class="Icons">
+                    <a href="#https://www.linkedin.com/in/tite-lunda-094956199/"><img src="./Images/Linkedin icon.png"
+                            alt="Linkedin"></a>
+                </li>
+                <li class="Icons">
+                    <a href=""><img src="./Images/angellist.png" alt="gmail"></a>
+                </li>
+                <li class="Icons">
+                    <a href="#https://twitter.com/LundaTite"><img src="./Images/twitter.png" alt="github"></a>
+                </li>
+                <li class="Icons">
+                    <a href="#https://github.com/TiteLunda"><img src="./Images/medium.png" alt="angellist"></a>
+                </li>
+
+            </ul>

```

Comment:
Well done so far, your project is taking shape. However, there a few things you need to work on to move to the next level. Kindly ensure that your design matches that of the figma design at all screen widths of the desktop version ie: 768px and beyond. There are a few differences that needs to be worked on as shown below :point_down: 

- [ ] The background image of the headline and its contents (the title) seems to be cramped. 
- [ ] The distance between the headline section and the project/works section is bigger than the actual design. 
Kindly check and fix the above issues so that your design will match the actual design 

### Your design @ 780px (width)

![Screenshot from 2023-04-07 16-20-55](https://user-images.githubusercontent.com/65233860/230644438-838f04ad-1879-477f-8f0b-bf73865ba9de.png)


### Your design @1110px (width)

![Screenshot from 2023-04-07 16-34-29](https://user-images.githubusercontent.com/65233860/230644531-a0d6005c-e090-4a76-b65d-a88a58d4dd40.png)


### Actual desktop design 

![Screenshot from 2023-04-07 16-24-51](https://user-images.githubusercontent.com/65233860/230644642-ea03f809-4a91-4a2e-86d4-055be1dac2e8.png)